net/crowdsec-firewall-bouncer/files/crowdsec-firewall-bouncer.defaults

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+# Default firewall backend is iptables instead of nftables
+FW_BACKEND="iptables"
+API_KEY=""
+CONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+
+## CheckFirewall
+iptables="true"
+which iptables > /dev/null
+FW_BACKEND=""
+if [[ $? != 0 ]]; then
+  echo "iptables is not present"
+  iptables="false"
+else
+  FW_BACKEND="iptables"
+  echo "iptables found"
+fi
+
+nftables="true"
+which nft > /dev/null
+if [[ $? != 0 ]]; then
+  echo "nftables is not present"
+  nftables="false"
+else
+  FW_BACKEND="nftables"
+  echo "nftables found"
+fi
+
+if [ "$nftables" = "true" -a "$iptables" = "true" ]; then
+  echo "Found nftables(default) and iptables..."
+fi
+
+if [ "$FW_BACKEND" = "iptables" ]; then
+  which ipset > /dev/null
+  if [[ $? != 0 ]]; then
+    echo "ipset not found, install it !"
+  fi
+fi
```

Comment:
Now that this backend detection is done in on every service start, this doesn't need to be here. You only need to set the backend value to `auto`. (Or just install a default config file with this value.)